difflib_test.go

```diff
@@ -282,5 +393,10 @@ func TestDiff(t *testing.T) {
 				i, seq1, seq2, ppDiff, test.PPDiff)
 		}
 
+		anchoredDiff := AnchoredDiff(seq1, seq2)
+		if !reflect.DeepEqual(anchoredDiff, test.AnchoredDiff) {
```

Comment:
Since we expect the same result between Diff and AnchoredDiff, we should only have one expectation.

Reply:
This is not true in some scenarios (I have added a new test for this)